code/modules/client/preference_setup/loadout/items/smoking.dm

```diff
@@ -144,3 +144,17 @@
 	ecig["ordinary electronic cigarette"] = /obj/item/clothing/mask/smokable/ecig/util
 	ecig["deluxe electronic cigarette"] = /obj/item/clothing/mask/smokable/ecig/deluxe
 	gear_tweaks += new /datum/gear_tweak/path(ecig)
+
+/datum/gear/smoking/cigarettecase //loadout list for cigarette cases. add new custom one's here
+	display_name = "cigarette cases selection"
+	description = "A selection of empty cigarette cases."
+	path = /obj/item/storage/box/fancy/cigarettes/case
+
+/datum/gear/smoking/cigarettecase/New()
+	..()
+	var/list/cigarettecase = list()
+	cigarettecase["Cigarette Case"] = /obj/item/storage/box/fancy/cigarettes/case
+	cigarettecase["Decorated Cigarette Case"] = /obj/item/storage/box/fancy/cigarettes/case/mus
+	cigarettecase["Solarian Cigarette Case"] = /obj/item/storage/box/fancy/cigarettes/case/sol
+	cigarettecase["Biesel Cigarette Case"] = /obj/item/storage/box/fancy/cigarettes/case/tc
```

Comment:
lower case for the names
"Cigarette Case" ---> "cigarette case

name first, then descriptor
"Decorated Cigarette Case" ---> "cigarette case, decorated"